spec/system/admin/order_cycles/simple_spec.rb

```diff
@@ -301,6 +317,18 @@
         click_button 'Add distributor'
         expect(page).to have_content "Permitted distributor"
 
+        within("tr.distributor-#{distributor_permitted.id}") { click_button 'Add fee' }
+        expect(page).to have_select(
+          "order_cycle_outgoing_exchange_1_enterprise_fees_0_enterprise_id", minimum: 1
+        )
```

Comment:
Is the `minimum` parameter necessary here? I would hope that we can predict exactly how many appear, and where. 
Actually, is this `expect` necessary at all? I think it would be covered by the next `select ... from:...` anyway.